source/features/more-dropdown.tsx

```diff
@@ -43,7 +43,7 @@ function onlyShowInDropdown(id: string): void {
 		return;
 	}
 
-	tabItem!.parentElement!.remove();
+	tabItem!.closest('.UnderlineNav-item')!.remove();
```

Comment:
They changed it again. Does this work? I'm still AFK. Please someone send a PR before v21.4.1 is released

```suggestion
	tabItem!.closest('.UnderlineNav-item, li')!.remove();
```